packages/@ember/-internals/metal/lib/events.ts

```diff
@@ -146,7 +146,7 @@ export function sendEvent(
     if (!target) {
       target = obj;
     }
-    if ('string' === typeof method) {
+    if (typeof method === 'string' || typeof method === 'symbol') {
```

Comment:
so I can, but I think we actually should do some digging with @krisselden here. We realized that VMs might do something special when they see `typeof foo` in an `if` statement specifically, but we did not confirm or look into it. And VMs might also be able to see that `typeof method` is being assigned and then used immediately and optimize that, and it's fewer bytes.

Completely unrelated, I think for now we should assume it would be two operations.

Reply:
Also note, this will mess with TypeScript a bit:

```js
let type = typeof method;
    if (type === 'string' || type === 'symbol') {
      method = target[method as string] as Function;
    }

    (method as Function).apply(target, params);
```

Reply:
OK, lets punt on this for now.